contrib/network-storage/heketi/heketi.yml

```diff
@@ -0,0 +1,10 @@
+---
+- hosts: heketi-node
+  roles:
+    - { role: prepare }
+
+- hosts: localhost
```

Comment:
This is causing me problems. It assumes localhost has direct access to the Kubernetes API, when it could be behind a bastion. Is it possible to use follow the approach in the glusterfs implementation which runs kubectl on one of the masters?
- https://github.com/kubernetes-incubator/kubespray/blob/v2.5.0/contrib/network-storage/glusterfs/glusterfs.yml#L22
- https://github.com/kubernetes-incubator/kubespray/blob/v2.5.0/contrib/network-storage/glusterfs/roles/kubernetes-pv/ansible/tasks/main.yaml#L11

Reply:
Ok I will make the master do the provisioning and also switch over to use the kube module.

Reply:
This will add additional complexity to the master nodes as they now have to have heketi-cli installed in order to configure heketi.

Reply:
Looks like I can solve it by using the heketi-cli executable shipped with the heketi pod. That actually reduces the complexity of the recipe because I do not need to setup kubectl port-forward stuff.